Container Security/kernel-eks-implementation-guide.md

```diff
@@ -0,0 +1,242 @@
+       -v ~/.aws:/root/.aws -it --rm \
+       quay.io/crowdstrike/cloud-tools-image
+   ```
+   The above command creates new container runtime that contains tools needed by this guide. All the
+   following commands should be run inside this container. If you have previously used AWS CLI tool,
+   you may already have AWS Credentials stored on your system in `~/.aws` directory. If that is the case,
+   it is preferential to start the container with `-v ~/.aws:/root/.aws:ro` option. This option should be
+   omitted if you don't want to share your AWS credentials with the container. You can review your
+   credentials with `aws sts get-caller-identity` command.
+
+    Example output
+    ```
+    {
+        "UserId": "AIDAXRCSSEFWMXXXXXXXX",
+        "Account": "123456789123",
+        "Arn": "arn:aws:iam::123456789123:user/xxxxxxx"
+    }
+    ````
+
+### Step 2: Create EKS Cluster
+
+ - Create new EKS cluster. It may take couple minutes before cluster is fully up and functioning.
+   ```
+   $ eksctl create cluster
+       --name demo-cluster --region eu-west-1 \
+       --managed
+   ```
+
+ - (optional) Verify that your local kubectl utility has been configured to connect to the cluster.
+   ```
+   $ kubectl cluster-info
+   Kubernetes control plane is running at https://EEAB38XXXXXXXXXXXXXXXXXXXXXXXXXX.sk1.eu-west-1.eks.amazonaws.com
+   CoreDNS is running at https://EEAB38XXXXXXXXXXXXXXXXXXXXXXXXXX.sk1.eu-west-1.eks.amazonaws.com/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+   To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+   ```
+   kubectl is command line tool that lets you control Kubernetes clusters. For configuration, kubectl looks for a file named config in the `$HOME/.kube` directory. This config was created previously by `eksctl create cluster` command and contains login information for your newly created cluster.
+
+### Step 3: Create ECR Container Repository
+
+ - Create container repository in AWS ECR. AWS ECR (Elastic Container Registry) is a cloud service providing a container registry. The bellow command creates a new repository in the registry and this repository will be subsequently used to store the Falcon Node sensor image.
+   ```
+   $ aws ecr create-repository --region eu-west-1 --repository-name falcon-node-sensor
+   {
+       "repository": {
+           "repositoryArn": "arn:aws:ecr:eu-west-1:123456789123:repository/falcon-node-sensor",
+           "registryId": "12345678912345",
+           "repositoryName": "falcon-sensor",
+           "repositoryUri": "123456789123.dkr.ecr.eu-west-1.amazonaws.com/falcon-node-sensor",
+           "createdAt": "2021-02-04T10:30:30+00:00",
+           "imageTagMutability": "MUTABLE",
+           "imageScanningConfiguration": {
+               "scanOnPush": false
+           },
+           "encryptionConfiguration": {
+               "encryptionType": "AES256"
+           }
+       }
+   }
+   ```
```

Comment:
```suggestion
   $ aws ecr create-repository --region eu-west-1 --repository-name falcon-node-sensor
   ```

  This will generate output similar to the following:
  ```
   {
       "repository": {
           "repositoryArn": "arn:aws:ecr:eu-west-1:123456789123:repository/falcon-node-sensor",
           "registryId": "12345678912345",
           "repositoryName": "falcon-sensor",
           "repositoryUri": "123456789123.dkr.ecr.eu-west-1.amazonaws.com/falcon-node-sensor",
           "createdAt": "2021-02-04T10:30:30+00:00",
           "imageTagMutability": "MUTABLE",
           "imageScanningConfiguration": {
               "scanOnPush": false
           },
           "encryptionConfiguration": {
               "encryptionType": "AES256"
           }
       }
   }
   ```
```

Reply:
Goal here is to clearly show what the user should copy/paste, vs what the output should look like